R/fit.R

```diff
@@ -359,6 +360,8 @@ mmrm <- function(formula,
   if (control$method %in% c("Kenward-Roger", "Kenward-Roger-Linear") && !reml) {
     stop("Kenward-Roger only works for REML")
   }
+  # TODO insert informative error message if cov struc is non-spatial and time point variables is not factor
```

Comment:
```suggestion
```

Reply:
This is a silly oversight on my part -- text removed.